src/Components/OATModelList/OATModelList.tsx

```diff
@@ -102,17 +102,20 @@ const OATModelList = ({
 
     const onModelDelete = (id) => {
         if (!modified) {
+            const dispatchDelete = () => {
+                dispatch({
+                    type: SET_OAT_PROPERTY_EDITOR_MODEL,
+                    payload: null
+                });
+                dispatch({
+                    type: SET_OAT_DELETED_MODEL_ID,
+                    payload: id
+                });
```

Comment:
Let's make the SET_OAT_DELETED_MODEL_ID dispatch set the property editor model state to null. So we don't need to do two dispatches here.

Reply:
Done